exer_7.py

```diff
@@ -0,0 +1,13 @@
+import re
+emails = ['abc@gmail.com','123$tt*@xyz.com', 'good@bad@uk.in','nasa@usa12.space','no-reply@domain.in','ramhanuman@saketa.lok','ruhi.mohan@exter123.123','fake@fake123.fakercom']
+
+validEmails = []
+emailPattern =  '[a-zA-Z0-9_-]+@[a-zA-Z0-9]+\.[a-zA-Z]{2,3}$'
+ 
+
+for email  in emails:
```

Comment:
Make sure to not have extra spaces in code. After `email`, there are 2 spaces, but it should be one only. Please remove that extra sapce.